core/templates/pages/exploration-editor-page/modal-templates/lost-changes-modal.controller.ts

```diff
@@ -19,6 +19,9 @@
 require(
   'components/common-layout-directives/common-elements/' +
   'confirm-or-cancel-modal.controller.ts');
+require(
```

Comment:
Wasn't the directive imported here?

Reply:
No, it wasn't. I've added it because the modal was breaking.